test/libsolidity/syntaxTests/inlineAssembly/shadowing/variable_by_opcode.sol

```diff
@@ -9,6 +9,5 @@ contract C {
     }
 }
 // ----
-// Warning 8261: (109-119): Variable is shadowed in inline assembly by an instruction of the same name
```

Comment:
Why don't we warn here?

Reply:
This warning was added when inline assembly looked quite different. It was a mixture of assembly and yul, i.e. allowed "functional" and "non-functional" syntax. Example: `1 mynumber add` was valid.

Today we have a specific error when a function is assigned but not executed. As a result this warning is not that useful as `let a := mload` would raise an error already.